src/components/table/table.tsx

```diff
@@ -593,10 +591,16 @@ export class Table {
     };
 
     private handleMoveColumn = (_, components: Tabulator.ColumnComponent[]) => {
-        const columns = components.map(this.findColumn);
+        const columns = this.findColumns(components);
         this.changeColumns.emit(columns);
     };
 
+    private findColumns = (
```

Comment:
I think that if a function starts with find in its name it involves some kind of search. In this case the function should just return all columns that is not undefined. Therefore, I think getColumns would be a better name 🤔 What do you think?